src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -122,9 +122,11 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       ...((bordered || v.borderColor || circular) &&
         getBorderedStyles(circular, v.borderColor || getIconColor(color, v))),
 
-      ...(rtl && {
-        transform: `scaleX(-1) rotate(${-1 * rotate}deg)`,
-      }),
+      ...(rtl &&
+        !isFontBased &&
```

Comment:
we may safely omit this `isFontBased` check, as there is an explicit flag required for those icons that need to be flipped